src/mscorlib/src/System/IO/DriveNotFoundException.cs

```diff
@@ -18,7 +18,7 @@ namespace System.IO {
     //Thrown when trying to access a drive that is not availabe.
     [Serializable]
     [System.Runtime.InteropServices.ComVisible(true)]
-    public class DriveNotFoundException : IOException {
+    internal class DriveNotFoundException : IOException {
```

Comment:
Where is this exception still used?  If it may be thrown to user code, it needs its deserialization ctor put back.  But I would have expected this whole type could be deleted from CoreCLR.